content/docs/2.7/scalers/rabbitmq-queue.md

```diff
@@ -32,6 +33,7 @@ triggers:
 - `queueName` - Name of the queue to read message from.
 - `mode` - QueueLength to trigger on number of messages in the queue. MessageRate to trigger on the publish rate into the queue. (Values: `QueueLength`, `MessageRate`)
 - `value` - Message backlog or Publish/sec. rate to trigger on.
+- `activationValue` - Min value to activate the scaler. (Default: `0`, Optional)
```

Comment:
"Activate the scaler" could be confusing, I guess, WDYT about `Message backlog or Publish/sec. rate to scale from/to zero`?